frontend/src/components/formsContainer/FormsContainer.js

```diff
@@ -0,0 +1,20 @@
+import CharacterForm from '../forms/CharacterForm.js'
+import StyleForm from '../forms/StyleForm.js'
+import GenreForm from '../forms/GenreForm.js'
+import LocationForm from '../forms/LocationForm.js'
+import "./FormsContainer.css";
+
+const FormsContainer = () => {
+  return (
+    <div className="forms-container">
+      <h2>Select a few details to get started</h2>
+      <CharacterForm />
+      <StyleForm />
+      <GenreForm />
+      <LocationForm />
```

Comment:
based on the comments above, these forms can now look like:

<DropdownForm dropdownItems={['rapunzel','spiderman','ironman'... etc]} selectionField={'character'}>
<DropdownForm dropdownItems={['medieval,'cyberpunk'...]} selectionField={'genre'}>
and so on